### ISSUE TITLE
Datasets

### ISSUE THREAD

**Issue by Andreyao** [RuijieJ/pren#4]
How could get the training datasets and testing datasets? Please.

**Comment by becxer** [comment]
Hi, do I have to convert manually from LMDB to image/txt files? 
Or could you provide the exact datasets to run the code?

**Comment by RuijieJ** [comment]
> 

hi, if you want to run our code directly, you need to convert LMDB to image and txt files.

Or you can modify dataset.py, and read images and labels from LMDB, e.g., you may use the "lmdb" python lib.

Since the training set is too large, I cannot upload it. Sorry for that.

**Comment by becxer** [comment]
Thank you!

**Comment by Chang-shuaishuai** [comment]
你好，请问一下如果训练中文模型的话，它的数据集也需要 'sample' 和 ' .txt ' 这种吗？那它的 'alphabet.txt '这个文件该怎么做呢?

**Comment by RuijieJ** [comment]
> 你好，请问一下如果训练中文模型的话，它的数据集也需要 'sample' 和 ' .txt ' 这种吗？那它的 'alphabet.txt '这个文件该怎么做呢?

数据集格式以及alphabet.txt和英文都是一样的，alphabet.txt里加入中文字符就行

**Comment by Chang-shuaishuai** [comment]
哦哦，谢谢大佬，救命恩人呢

**Comment by icecream-Tnak** [comment]
Thanks for your quality work. I would like to confirm that the results reported in the paper are the ICDAR 2013 857 and ICDAR 2015 1811 versions of the corresponding dataset. Because IC 13 and IC 15 both have two versions.

**Comment by qyfff** [comment]
![image](https://user-images.githubusercontent.com/103289265/186896556-cbf8607a-a4e9-4d02-91bf-efac68037242.png)

**Comment by qyfff** [comment]
代码有地方要改吗

**Comment by qyfff** [comment]
有哥哥有数据集？

**Comment by xushuanglong** [comment]
请问大佬train_list里面具体应该怎么写，我写了图片及标签路径报语法错误，有没有train_list内容例子呀

**Comment by saitejabandaru** [comment]
> Does anyone have a data set?

Yes please anyone having the exact dataset using in this paper??
